### PR TITLE
Fixed `this` bindings and access to `config`

### PR DESCRIPTION
**Pull request by MadMG**
Class methods in javascript are not bound by default.

please refer: [stackoverflow.com](https://stackoverflow.com/questions/43026563/getting-this-of-an-es6-class-to-call-methods-and-get-data-members)

### CODE REVIEW

File: kraken.js [hunk 1]
```diff
@@ -32,6 +32,8 @@ module.exports = class KrakenClient {
 			otp: options.otp,
 			timeoutMS: options.timeout || 5000
 		};
+
+    this.api = this.api.bind(this);
```
Comment (Mik13) [inline kraken.js]: Could you please use tabs instead of spaces?

**Comment by Mik13** [comment]
Good catch!

Could you please look at my comment? I would be very happy to merge that afterwards.

**Comment by MadMG** [comment]
fixed!

i suggest to add a .editorconfig file :)

**Comment by Mik13** [comment]
Perfect, thank you very much!

**Pull request merged**